### PR TITLE
Business plan -> Business Cloud rename

### PR DESCRIPTION
**Pull request by bluemazzoo**
Minor naming changes throughout terms: "GitHub Business plan" renamed to "GitHub Business Cloud", as documented on the pricing page https://github.com/pricing/business-cloud
- No other changes, except for a letter correction in Q(2) of the Corporate Terms of Service
- Other changes in Business Cloud Addendum are for grammar and flow

### CODE REVIEW

**Review by nsqe — APPROVED** [review]
If I _must_.... :trollface:

File: Policies/github-business-cloud-addendum.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 ---
-title: GitHub Business Plan Addendum
+title: GitHub Business Cloud Addendum
 ---
 
-### Business Plan Addendum
```
Comment (Mistydjones007) [inline Policies/github-business-cloud-addendum.md:5]: Suck a penis Jake Hudson was

**Review by Hecim1984 — COMMENTED** [review]
🎁🚀

**Review by Altamiranohj — COMMENTED** [review]
@@.Gracias por el aporte#(2)#491

**Comment by Altamiranohj** [comment]
Thanks you good bye..!!

**Comment by Altamiranohj** [comment]
Muchas Gracias encerio..

File: Policies/github-business-cloud-addendum.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+### Business Cloud Addendum
 
-These are the additional terms for GitHub's Business plan ("Business Plan Terms"). Any capitalized terms not defined in these Business Plan Terms will be defined in the [Terms of Service](/articles/github-terms-of-service/) or the [Corporate Terms of Service](/articles/github-corporate-terms-of-service/). By agreeing to these Business Plan Terms, you also agree to the Terms of Service, as determined by the type of account you have.
+These are the additional terms for GitHub's Business Cloud ("Business Cloud Terms"). Any capitalized terms not defined in these Business Cloud Terms will be defined in the [Terms of Service](/articles/github-terms-of-service/) or the [Corporate Terms of Service](/articles/github-corporate-terms-of-service/). By agreeing to these Business Cloud Terms, you also agree to the Terms of Service, as determined by the type of account you have.
```
Comment (Bmg00) [inline Policies/github-business-cloud-addendum.md:7]:
```suggestion
These are the additional terms for GitHub's Business Cloud ("Business Cloud Terms"). Any capitalized terms not defined in these Business Cloud Terms will be defined in the [Terms of Service](/articles/github-terms-of-service/) or the [Corporate Terms of Service](/articles/github-corporate-terms-of-service/). By agreeing to these Business Cloud Terms, you also agree to the Terms of Service, as determined by the type of account you have.
[]()Bmg00
- 
```

**Review by Bmg00 — APPROVED** [review]
**@brycegerhardt2@gmail.com
**

**Comment by Bmg00** [comment]
> 🎁🚀

- [ ]Policies/github-terms-of-service.md

File: Policies/github-business-cloud-addendum.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
 ---
-title: GitHub Business Plan Addendum
+title: GitHub Business Cloud Addendum
```
Comment (Bezzo100-lab) [inline Policies/github-business-cloud-addendum.md:2]: # bezzoxoxo_

**Review by Bezzo100-lab — APPROVED** [review]

**Review by Turki53341 — COMMENTED** [review]
#- []()

File: Policies/github-business-cloud-addendum.md (hunk 3 above)
Comment (937-a11y) [reply Policies/github-business-cloud-addendum.md:2]: > bezzoxoxo_

**Review by 937-a11y — COMMENTED** [review]
****

**Comment by QiSHUXING** [comment]
01ad003300430030003800300034003000370042004100300030

File: Policies/github-business-cloud-addendum.md (hunk 3 above)
Comment (tintalaclave-tech) [inline Policies/github-business-cloud-addendum.md]: 🤟🏼🤟🏼

**Review by jshscvs-source — APPROVED** [review]

**Comment by QiSHUXING** [comment]
你好。已收到你的邮件，我会在假期结束后尽快处理。————————————这是来自QQ邮箱的自动回复邮件。

**Review by jshscvs-source — APPROVED** [review]

**Pull request merged**